bots/task/testmap.py

```diff
@@ -0,0 +1,125 @@
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-6',
+        ],
+        'rhel-7.7': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-7',
+        ],
+        'rhel-8.0': ['avocado/fedora', 'container/bastion', 'selenium/firefox', 'selenium/chrome',
+            'verify/rhel-8-0',
+        ],
+        'rhel-8-appstream': ['avocado/fedora', 'container/bastion', 'selenium/firefox',
+            'selenium/chrome', 'verify/rhel-8-0-distropkg', 'verify/rhel-8-1',
+        ],
+        'rhel-8.1': ['verify/rhel-8-1',
+        ],
+        # These can be triggered manually with bots/tests-trigger
+        '_manual': ['verify/fedora-i386', 'verify/fedora-testing',
+        ],
+    },
+    'cockpit-project/starter-kit': {
+        'master': [
+            'cockpit/centos-7',
+            'cockpit/fedora-30',
+        ],
+    },
+    'cockpit-project/cockpit-ostree': {
+        'master': [
+            'cockpit/fedora-atomic',
+            'cockpit/continuous-atomic',
+            'cockpit/rhel-atomic',
+        ],
+    },
+    'cockpit-project/cockpit-podman': {
+        'master': [
+            'cockpit/fedora-29',
+            'cockpit/fedora-30',
+            'cockpit/rhel-8-1',
+        ],
+    },
+    'weldr/lorax': {
+        'master': [
+            'cockpit/fedora-30',
+            'cockpit/fedora-30/live-iso',
+            'cockpit/fedora-30/qcow2',
+            'cockpit/fedora-30/aws',
+            'cockpit/fedora-30/openstack',
+            'cockpit/fedora-30/vmware',
+        ],
+        '_manual': [
+            'cockpit/fedora-30/azure',
+
+            'cockpit/rhel-8-1',
+            'cockpit/rhel-8-1/live-iso',
+            'cockpit/rhel-8-1/qcow2',
+            'cockpit/rhel-8-1/aws',
+            'cockpit/rhel-8-1/azure',
+            'cockpit/rhel-8-1/openstack',
+            'cockpit/rhel-8-1/vmware',
+        ],
+        'rhel7-extras': [
+            'cockpit/rhel-7-7',
+            'cockpit/rhel-7-7/live-iso',
+            'cockpit/rhel-7-7/qcow2',
+            'cockpit/rhel-7-7/aws',
+            'cockpit/rhel-7-7/azure',
+            'cockpit/rhel-7-7/openstack',
+            'cockpit/rhel-7-7/vmware',
+        ],
+    },
+    'weldr/cockpit-composer': {
+        'master': [
+            'cockpit/fedora-30/chrome',
+            'cockpit/fedora-30/firefox',
+            'cockpit/fedora-30/edge',
+            'cockpit/rhel-7-7/firefox',
+            'cockpit/rhel-8-1/chrome',
+        ],
+        'rhel-8.0': ['cockpit/rhel-8-0/chrome', 'cockpit/rhel-8-0/firefox', 'cockpit/rhel-8-0/edge'
+        ],
+        'rhel-8.1': ['cockpit/rhel-8-1/chrome', 'cockpit/rhel-8-1/firefox', 'cockpit/rhel-8-1/edge'
+        ],
+    },
+    'mvollmer/subscription-manager': {
+        '_manual': [
+            'cockpit/rhel-8-0',
+        ],
+    }
+}
+
+def projects():
+    return REPO_BRANCH_CONTEXT.keys()
```

Comment:
I see that we unfortunately still need this for triggering external tests for bots/ changes. That will be the really difficult bit in the future, as there needs to be one central place that knows all projects that we run CI for. But let's keep this for now indeed.